src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +328,102 @@ public void SetFocus(bool focused)
             Interop.ChromiumEwk.ewk_view_focus_set(_realHandle, focused);
         }
 
+        /// <summary>
+        /// Get coordinates of content.
+        /// </summary>
+        /// <param name="width">The width of the coordinate.</param>
+        /// <param name="height">The height of the coordinate.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool GetContentsSize (ref int width, ref int height)
+        {
+            return Interop.ChromiumEwk.ewk_view_contents_size_get(_realHandle, ref width, ref height);
+        }
+
+        /// <summary>
+        /// Get cookies for URL.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <returns>'cookie string' on success, otherwise 'null'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public string GetCookies (string url)
+        {
+            return Interop.ChromiumEwk.ewk_view_get_cookies_for_url(_realHandle, url);
+        }
+
+        /// <summary>
+        /// Exit full screen.
+        /// </summary>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool ExitFullscreen ()
+        {
+            return Interop.ChromiumEwk.ewk_view_fullscreen_exit(_realHandle);
+        }
+
+        /// <summary>
+        /// Gets the current load progress of the page.
+        /// </summary>
+        /// <returns>'value 0.0 to 1.0' on success, otherwise '-1.0'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public double GetLoadAProgress ()
+        {
+            return Interop.ChromiumEwk.ewk_view_load_progress_get(_realHandle);
+        }
+
+        /// <summary>
+        /// Sends the orientation of the device.
+        /// </summary>
+        /// <param name="orientation">The new orientation of the device in degree.</param>
```

Comment:
Need more detail for the orientation. And how about defining it as enum?

Reply:
Done